### PR TITLE
Add erik's operation.yml stuff

### PR DESCRIPTION
**Pull request by erik-whiting**
The rest of my data for https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2229

### CODE REVIEW

File: _data/operations.yml [hunk 1]
```diff
@@ -314,3 +314,12 @@ members:
...
   wikidata: Q108555457
+- affiliation: University of Nebraska, Lincoln, NE
+  country: USA
+  github: erik-whiting
+  groups:
+  - technical
+  name: Erik Whiting
+  orcid: 0000-0003-1022-5281
+  ror: 043mer456
+  wikidata: Q1353679
```
[the remark this answers is not in the document]
Comment (cthoyt) [reply _data/operations.yml:325]: here's a guide on how to do this well: https://cthoyt.com/2021/08/17/self-organization.html

**Comment by erik-whiting** [comment]
Is this failure because I don't have a picture?
```
_________________________ TestMembershipData.test_data _________________________

self = <test_memberships.TestMembershipData testMethod=test_data>

    def test_data(self):
        """Test the working group data is clean."""
>       res = Group.parse_obj(yaml.safe_load(OPERATIONS_METADATA_PATH.read_text()))

tests/test_memberships.py:50: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pydantic/main.py:526: in pydantic.main.BaseModel.parse_obj
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   pydantic.error_wrappers.ValidationError: 1 validation error for Group
E   members -> 33 -> affiliation
E     value is not a valid dict (type=type_error.dict)

pydantic/main.py:342: ValidationError
=========================== short test summary info ============================
FAILED tests/test_memberships.py::TestMembershipData::test_data - pydantic.er...
======================== 1 failed, 10 passed in [41](https://github.com/OBOFoundry/OBOFoundry.github.io/actions/runs/3708519335/jobs/6286163045#step:5:42).19s =========================
ERROR: InvocationError for command /home/runner/work/OBOFoundry.github.io/OBOFoundry.github.io/.tox/py/bin/pytest tests/ (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py: commands failed
make: *** [Makefile:141: tox] Error 1
```

**Comment by cthoyt** [comment]
You’re going to have to change your entry to match the new schema, see any of the others on the main branch as an example

**Comment by cthoyt** [comment]
I’d suggest you also add a picture before finishing this PR id you haven’t already since the other #2249 will start enforcing it

**Comment by erik-whiting** [comment]
@cthoyt I added a picture of myself in this PR: https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2233
do I need to add it in another spot?

**Comment by cthoyt** [comment]
Nope, once was enough!

**Comment by erik-whiting** [comment]
ok whoever's reviewing this, this is ready for another pass @rays22 @nicolevasilevsky

**Comment by erik-whiting** [comment]
> Looks good!

omg I'm so sorry I mean to tag @matentzn

**Review by rays22 — APPROVED** [review]

**Pull request merged**